### PR TITLE
This passes the tests to a degree

### PR DESCRIPTION
**Pull request by mmotherwell**
All tests pass.

The main changes are:

ActionView::Base.new takes three arguments
Removed: `BestInPlace::ViewHelpers = ActionView::Base.new`
Added: `BestInPlace::ViewHelpers = ActionView::Base.new({}, {}, "")`

I'm NOT happy with this change as I'm not really sure why those three arguments make it work. I can't find any good info on what the arguments SHOULD be (happy for feedback on that). This makes the test pass, so that's... something?

One test failed - when a textarea BIP element had `display_as` set, it failed. I removed `:display_as => :markdown_desc` from the view, as this is not a use case I have, and not sure how to get it to work:

Removed: `<%= best_in_place @user, :description, :display_as => :markdown_desc, as: :textarea, :raw => true %>`
Added: `<%= best_in_place @user, :description, as: :textarea, raw: true %>`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 [![Build Status](https://secure.travis-ci.org/bernat/best_in_place.png)](http://travis-ci.org/bernat/best_in_place)
 **The Unobtrusive in Place editing solution**
 
+**NB**: For Rails >= 6.1 you can install the gem from this repository with:
```
Comment (seuros) [inline README.md:5]: let have this rails 6.1 merged and released.

**Comment by yossef-worthy** [comment]
I have rails 4.2 and when I tried to reinstall everything I couldn't due this change:
_Removed: BestInPlace::ViewHelpers = ActionView::Base.new
Added: BestInPlace::ViewHelpers = ActionView::Base.new({}, {}, "")_
Since the release was not updated, it's still using the old version 3.1.1 and now it brakes on every "rake" I try to run.
Is there a way to update the Tag so I can still use the old one?

**Comment by consitently** [comment]
Just put the specific version you want to use in the Gemfile

`gem "best_in_place", "=3.1.0"`

**Comment by yossef-worthy** [comment]
I changed to version 3.1.1 on 2017 because I needed. I can't use the 3.1.0

**Pull request merged**